mssqlcli/mssqlcliclient.py

```diff
@@ -99,11 +99,15 @@ def connect(self):
             u'connection_request', connection_params, self.owner_uri)
         connection_request.execute()
 
+        error_messages = []
         while not connection_request.completed():
             response = connection_request.get_response()
-            if response:
-                response = connectionservice.handle_connection_response(
-                    response)
+
+            if isinstance(response, connectionservice.ConnectionCompleteEvent):
+                if response.error_message:
+                    error_messages.append(u'Error message: {}'.format(response.error_message))
+                if response.messages:
+                    error_messages.append(u'Additional message: {}'.format(response.messages))
```

Comment:
Do not add additional messages. This has the entire call stack of SqlClient. That is why previously we just output the error and only logged the message.

Reply:
updated.